tests/apis.js

```diff
@@ -36,96 +35,319 @@ test.cb.before(t => {
   });
 });
 
-test.cb.after.always(t => {
-  const items = types.map(type => {
-    return database('live').where('type', type).del().then(() => {
-      return database('schedule').where('type', type).del();
-    });
+//////////////////////////////
+// Utils - attributes
+//////////////////////////////
+test.serial.skip('Utils: attributes - empty query', t => {
+  const random = Math.round(Math.random() * (live.length - 1));
+  let expected = live[random];
+  if (expected === undefined) {
+    expected = cloneDeep(live[(live.length - 1)]);
+  }
+
+  const model = allTypes.find(typ => {
+    return typ.id === expected['type-slug'];
   });
 
-  Promise.all(items)
-    .then(() => {
-      t.end();
-    })
-    .catch(e => {
-      t.fail(e);
-    });
+  const query = {};
+
+  const attributes = apiUtils.attributes(expected.attributes, model.attributes, allTypes, query);
+
+  return attributes.then(result => {
+    t.is(typeof result, 'object', 'Should contain result, an object.');
+    t.is(Object.keys(result).length, 6, 'Should contain six main objects.');
```

Comment:
The fact that it goes on to test with `utils.referencer` doesn't negate the fact that these tests can easily result in false positives